lib/crypto/crypto_keys_subr.c

```diff
@@ -180,23 +183,23 @@ crypto_keys_subr_import_RSA_priv(RSA ** key, const uint8_t * buf, size_t buflen)
 	if (import_BN(&n, &buf, &buflen))
 		goto err1;
```

Comment:
Double-checking BN_free(NULL)... the docs for 1.1 says it's fine [1], but the docs for 1.0.2 doesn't mention it [2].  However, the code for 1.0.2 [3] and even 0.9.6 [4] shows that it's ok.  Should I do it manually anyway, just to be safe?

[1] https://www.openssl.org/docs/manmaster/man3/BN_free.html
[2] https://www.openssl.org/docs/man1.0.2/crypto/BN_free.html
[3] https://github.com/openssl/openssl/blob/OpenSSL_1_0_2-stable/crypto/bn/bn_lib.c#L247
[4] https://github.com/openssl/openssl/blob/OpenSSL_0_9_6-stable/crypto/bn/bn_lib.c#L278